README.md

```diff
@@ -2160,6 +2169,16 @@ set :protection, :session => true
       <tt>set :static_cache_control, [:public, :max_age => 300]</tt>
     </dd>
 
+  <dt>static_headers</dt>
+    <dd>
+      Allows you to define custom header settings for static file responses. Header keys should be lowercase (as required by Rack 3).
```

Comment:
Actually, this is not strictly needed here. They will be lowercased by Rack, via [`Rack::Response#headers`](https://github.com/rack/rack/blob/v3.1.13/lib/rack/response.rb#L61-L65) (which is an instance of [`Rack::Headers`](https://github.com/rack/rack/blob/v3.1.13/lib/rack/headers.rb#L3-L8)).

```suggestion
      Allows you to define custom header settings for static file responses.
```

Reply:
I was mostly thinking we want all our examples (README and test) to use lowercase to encourage that style going forward.